src/open_inwoner/openklant/wrap.py

```diff
@@ -95,3 +96,27 @@ def get_fetch_parameters(request, use_vestigingsnummer: bool = False) -> dict:
                 parameters.update({"vestigingsnummer": vestigingsnummer})
         return parameters
     return {}
+
+
+def get_local_kcm_mapping(
+    kcms: list[KlantContactMoment], user: User
+) -> dict[str, KlantContactMomentLocal]:
+    existing_kcms = KlantContactMomentLocal.objects.filter(user=user).values_list(
+        "kcm_url", flat=True
+    )
```

Comment:
I think this should to be wrapped in a `set()`? Otherwise you're doing `in` into a queryset.